sdk/documentintelligence/ai-document-intelligence-rest/src/outputModels.ts

```diff
@@ -146,6 +148,16 @@ export interface DocumentFieldSchemaOutput {
   properties?: Record<string, DocumentFieldSchemaOutput>;
 }
 
+/** The error object. */
+export interface WarningOutput {
```

Comment:
We have this as `Warning` in Java, why Output?

Reply:
All the interfaces from previous beta also have Output suffix in them, so this is in sync with them.
I'll check with Architects after this beta, incase we'd want to revert them all.